### DISCUSSION TITLE
how to accelerate through SSD?

### DISCUSSION THREAD

**Discussion by timchenxiaoyu** [seaweedfs/seaweedfs#2340] [General]
how to accelerate through SSD?

**Comment by timchenxiaoyu** [comment]
server contain SSD and normal SAS disk,how to configuration

**Comment by chrislusf** [comment]
Tiered storage will be provided later. The plan is to periodically check and balance the load by moving volumes to different storages.

**Comment by wangyongqiang0617** [comment]
i start 12 volume server, one server with a SATA disk
use seaweed benchmark, write parallel with 100 threads, write 2M files
i see: only 4 or 5 disk has pressure, not all 12 disk

how to make data write more balance, i think this will improve performance

**Comment by chrislusf** [comment]
@wangyongqiang0617 https://github.com/chrislusf/seaweedfs/wiki/Optimization#increase-concurrent-writes

**Comment by onlyjob** [comment]
A good idea is to write mostly to SSD and eventually move data to rotational disks.

**Comment by ekozlov-search** [comment]
Now you can mark volume servers as SSD or HDD using -dataCenter paramater. Upload data to SSD using "/dir/assign?dataCenter=SSD". And eventualy move volume from SSD to HDD server when you need using commad "shell volume.move". I hope, sometime we will have this function out of the box.

**Comment by chrislusf** [comment]
This is related to a fixed issue https://github.com/chrislusf/seaweedfs/issues/1265. 

In 2.13, the volume server index storage can be configured to smaller faster SSD.

Other SSD acceleration will be worked on later.

**Comment by taozix** [comment]
bcache is a choice.

**Comment by onlyjob** [comment]
Bcache is rubbish. [dm-writeboost](https://github.com/akiradeveloper/dm-writeboost) is better.
